cpplint/cpplint.py

```diff
@@ -4291,6 +4292,15 @@ def GetLineWidth(line):
       if unicodedata.east_asian_width(uc) in ('W', 'F'):
         width += 2
       elif not unicodedata.combining(uc):
+        # Issue 337        
+        if (sys.version_info.major, sys.version_info.minor) == (2, 7) :
+          # https://github.com/python/cpython/blob/2.7/Include/unicodeobject.h#L81
+          is_wide_build = sysconfig.get_config_var("Py_UNICODE_SIZE") >= 4
+          # https://github.com/python/cpython/blob/2.7/Objects/unicodeobject.c#L564
+          is_low_surrogate = (0xDC00 <= ord(uc) and ord(uc) <= 0xDFFF)
```

Comment:
I believe this can be written as `0xDC00 <= ord(uc) <= 0xDFFF`

Reply:
Thanks!